Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -0,0 +1,92 @@
+                    constraints.append(button.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -config.layoutUIOffset.vertical))
+                }
+                if config.location.contains(.top) {
+                    constraints.append(button.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: config.layoutUIOffset.vertical))
+                }
+                if config.location.contains(.right) {
+                    constraints.append(button.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor, constant: -config.layoutUIOffset.horizontal))
+                }
+                if config.location.contains(.left) {
+                    constraints.append(button.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: config.layoutUIOffset.horizontal))
+                }
+                NSLayoutConstraint.activate(constraints)
+            }
+            
+            required init?(coder: NSCoder) {
+                fatalError("init(coder:) has not been implemented")
+            }
+            
+            @objc func buttonPressed() {
+                if config.animations {
+                    UIView.animate(withDuration: 0.3, delay: 0, options: .curveEaseInOut) {
+                        self.button.alpha = 0
+                    }
+                } else {
+                    button.isHidden = true
+                }
+                let formDialog = UIViewController(nibName: nil, bundle: nil)
+                formDialog.view.backgroundColor = .white
+                let label = UILabel(frame: .zero)
+                label.text = "Hi, I'm a user feedback form!"
+                formDialog.view.addSubview(label)
+                label.translatesAutoresizingMaskIntoConstraints = false
+                label.textAlignment = .center
+                NSLayoutConstraint.activate([
+                    label.leadingAnchor.constraint(equalTo: formDialog.view.leadingAnchor),
+                    label.trailingAnchor.constraint(equalTo: formDialog.view.trailingAnchor),
+                    label.centerYAnchor.constraint(equalTo: formDialog.view.centerYAnchor),
+                ])
+                present(formDialog, animated: true)
+            }
+        }
+        
+        init(config: SentryUserFeedbackWidgetConfiguration) {
+            super.init(frame: UIScreen.main.bounds)
+            rootViewController = RootViewController(config: config)
+            windowLevel = config.windowLevel
+        }
+        
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+        }
+        
+        override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+            guard let result = super.hitTest(point, with: event) else {
+                return nil
+            }
+            guard result.isKind(of: UIButton.self) else {
+                return nil
+            }
```

Comment:
l: ```suggestion
            guard let result = super.hitTest(point, with: event), !result.isKind(of: UIButton.self) else {
                return nil
            }
```

Reply:
Yes, I'll have to add a boolean property there to allow hits to the form while it's displayed. That'll come in a separate PR for implementing the form (and I have implementations of things I've done this for in the past that I can reference)

Reply:
I usually prefer leaving separate conditions in separate guards instead of using comma-delimited lists, helps debugging and logging later if need be